CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing
+
+Thanks you for your interest in contributing to Microsoft Caboodle! In this document we'll outline what you need to know about contributing and how to get started.
+
+## Code of Conduct
+
+Please see our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Prerequisite
+
+You will need to complete a Contribution License Agreement before any pull request can be accepted. Complete the CLA at https://cla2.dotnetfoundation.org/.
+
+## Contributing Code
+
+Check out [A Beginner's Guide for Contributing to Microsoft Caboodle](https://blog.xamarin.com/beginners-guide-contributing-xamarin-forms/).
+
+### Bug Fixes
+
+If you're looking for something to fix, please browse [open issues](https://github.com/xamarin/Caboodle/issues). 
+
+Follow the style used by the [.NET Foundation](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), with two primary exceptions:
```

Comment:
I also think we should not use `_variable` or `s_variable` as is their convention.  Thoughts?

Reply:
Agree. no _